### DISCUSSION TITLE
Why is the text color being hard-coded to white here?

### DISCUSSION THREAD

**Discussion by polarstoat** [WordPress/gutenberg#43539] [Q&A] [answered]
I've just had an issue of Buttons with the 'Fill' style having their text color overridden by this snippet here, what's the purpose of it? https://github.com/WordPress/gutenberg/blob/19c84206ca48ecc5a952fc088f7a3a15e60523a9/packages/block-library/src/button/style.scss#L18

It appears to take precedence over the text color which is set on `.wp-element-button, .wp-block-button__link`

**Comment by jasmussen** [comment, marked as answer]
I believe the color was necessary to ensure contrast against the default dark gray background that was also provided.

However it seems like those colors are now provided by a back-compat theme library, and can be removed here. I've created a PR to do that https://github.com/WordPress/gutenberg/pull/43553 — but would appreciate input there.

**Reply by jasmussen** [reply]
This has been fixed.
